payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AuBecsDebitMandateElementUI.kt

```diff
@@ -0,0 +1,97 @@
+package com.stripe.android.ui.core.elements
+
+import android.content.Intent
+import android.net.Uri
+import androidx.annotation.RestrictTo
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.stripe.android.ui.core.R
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun AuBecsDebitMandateElementUI(
+    element: AuBecsDebitMandateTextElement
+) {
+
+    val annotatedText = buildAnnotatedString {
+        val textColor = when {
+            element.color != null -> {
+                colorResource(element.color)
+            }
+            isSystemInDarkTheme() -> {
+                Color.LightGray
+            }
+            else -> {
+                Color.Black
+            }
+        }
+        val nonLinkTextStyle = SpanStyle(
+            color = textColor,
+            fontSize = element.fontSizeSp.sp,
+            letterSpacing = element.letterSpacingSp.sp,
+        )
+
+        withStyle(style = nonLinkTextStyle) {
+            append(stringResource(R.string.au_becs_mandate_pre_link))
+        }
+
+        // We attach this *URL* annotation to the following content
+        // until `pop()` is called
+        pushStringAnnotation(
+            tag = "URL",
+            annotation = "https://stripe.com/au-becs-dd-service-agreement/legal"
+        )
```

Comment:
As you mentioned this won't work with localized strings. If you do submit it like this please create a JIRA ticket and assign to me because I'll need to do something similar soon and can fix this too.